frontend/model/notifications/vuexModule.js

```diff
@@ -76,11 +76,9 @@ const mutations = {
     if (state.includes(notification)) {
       throw new Error('This notification is already in the store.')
     }
-    if (state[0] && age(notification) > age(state[0])) {
-      throw new Error('This notification is older than the latest one in the store.')
-    }
-    // Using `unshift()` here to insert the new item before older ones.
-    state.unshift(notification)
+    state.push(notification)
```

Comment:
One request - during my testing of a different PR where I changed the computer's clock, I got the `throw new Error('This notification is older than the latest one in the store.')` thrown, and this broke the app.

It turns out that we cannot throw errors here because these are being thrown inside of sideEffects, and if an error is thrown inside of a sideEffect it will stop the app completely from moving forward and doing anything. So errors thrown in sideEffects must be because something super bad happened.

So it's good that this PR removes that error, but there's still one on line 77. Can you change that to `console.error` + `return` instead?

Reply:
Ok, will do